source/codegen/metadata/nidcpower/config.py

```diff
@@ -1,29 +1,19 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 20.7.0d8
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DCPower API metadata version 22.8.0d9999
 config = {
     'additional_headers': {
         'custom/ivi_errors.h': [
             'service.cpp'
         ]
     },
-    'api_version': '20.7.0d8',
+    'api_version': '22.8.0d9999',
     'c_function_prefix': 'niDCPower_',
     'c_header': 'nidcpower.h',
     'close_function': 'Close',
-    'context_manager_name': {
-        'abort_function': 'Abort',
-        'initiate_function': 'Initiate',
-        'task': 'acquisition'
-    },
-    'csharp_namespace': 'NationalInstruments.Grpc.DCPower',
+    'csharp_namespace': 'NationalInstruments.Grpc.Dcpower',
```

Comment:
Should get this back to DCPower

Reply:
Have an [PR for the hapigen plugin](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/364704) to be able to set this.

Reply:
Fixed now.